mixer/cmd/server/cmd/server.go

```diff
@@ -168,11 +176,15 @@ func serverCmd(info map[string]template.Info, adapters []adptr.InfoFn, legacyAda
 
 	serverCmd.PersistentFlags().StringVarP(&sa.clientCertFiles, "clientCertFiles", "", "", "A set of comma-separated client X509 cert files")
 
-	// TODO: implement a better option to specify how traces are reported
-	serverCmd.PersistentFlags().StringVarP(&sa.traceOutput, "traceOutput", "t", "",
-		"If the literal string 'STDOUT' or 'STDERR', traces will be produced and written to stdout or stderr respectively. "+
-			"Otherwise the address is assumed to be a URL and HTTP zipkin traces are sent to that address. "+
-			"Note that when providing a URL it must be the full path to the span collection endpoint, e.g. 'http://zipkin:9411/api/v1/spans'.")
+	serverCmd.PersistentFlags().StringVarP(&sa.zipkinAddr, "zipkinAddr", "", "", "URL of zipkin collector (example: 'http://zipkin:9411/api/v1/spans'")
+	serverCmd.PersistentFlags().StringVarP(&sa.zipkinUser, "zipkinUser", "", "", "Username for zipkin (http basic auth)")
```

Comment:
We should use standard url format instead.
`zipkin://user:pass@host`

Reply:
fwiw, this is not how the backend libraries take the args. So this would mean additional parsing on our part.  Another potential consideration: USER and PASS mounted as secrets.  Thoughts?

Reply:
removed basic auth support altogether -- it was kinda ugly (prefer Istio for securing access). we can add it back if needed later.